wordpress-mobile/AztecEditor-iOS
# Issue/483 move processor

Fixes #483 

As discussed before, this PR moves the generic code for shortcode processing to the main project and keeps the specific WordPress Shortcode processor on the demo app.

This a pre-step to then implement shortcode on the main app.

To test:
 - Open the demo app
 - Check that all videos are still showing correctly.
 - Check if the tests are passing.
